otter_welcome_buddy/common/utils/dates.py

```diff
@@ -14,4 +14,4 @@ def get_current_month() -> int:
     @staticmethod
     def create_cron_trigger_from(crontab: str) -> CronTrigger:
         """Returns cron trigger from crontab"""
-        return CronTrigger.from_crontab(crontab)
+        return CronTrigger.from_crontab(crontab, "America/Mexico_City")
```

Comment:
Looks like this can be in a file just related to constants or configuration.

Reply:
You can add the timezone to the settings file that I suggested before as well.